package.json

```diff
@@ -13,8 +13,6 @@
   "files": [
     "build/protos",
```

Comment:
Going to make a guess - we probably need to include `build/proto` here as well.

Reply:
Ah right, good call!